CMakeLists.txt

```diff
@@ -591,14 +594,17 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 add_subdirectory(external/llvh)
 add_subdirectory(utils/hermes-lit)
-add_subdirectory(tools)
 add_subdirectory(include)
 add_subdirectory(lib)
 add_subdirectory(public)
 add_subdirectory(external)
 add_subdirectory(API)
 add_subdirectory(android/intltest/java/com/facebook/hermes/test)
 
+if(HERMES_ENABLE_TOOLS)
+  add_subdirectory(tools)
+endif()
```

Comment:
Looks like there are few changes in this diff that are unrelated styling changes? (apart from this one ofc)

Reply:
I’m unsure which changes you mean, can you elaborate?

Reply:
@grabbou I think you might be referring to all the whitespace changes below (?), which are because of the additional `if` check.